src/content/docs/en/guides/upgrade-to/v6.mdx

```diff
@@ -371,6 +371,38 @@ prefetch('/about');
 
 <ReadMore>Learn more about [prefetching](/en/guides/prefetch/).</ReadMore>
 
+### Removed: `rewrite()` from Actions context
+
+<SourcePR number="14477" title="feat!: remove rewrite from action context"/>
+
+In Astro 5.5.6, the `ActionAPIContext.rewrite` method was deprecated because custom endpoints should be used where you might want to use rewrites.
+
+Astro 6.0 removes the `rewrite()` method from `ActionAPIContext` entirely and it may no longer be used.
+
+#### What should I do?
+
+Review your Actions handlers and remove any call to `rewrite()`:
+
+
+```ts title="src/actions/index.ts" del={10}
+import { defineAction } from 'astro:actions';
+import { z } from 'astro:schema';
+
+export const server = {
+  getGreeting: defineAction({
+    input: z.object({
+      // ...
+    }),
+    handler: async (input, context) => {
+      context.rewrite('/')
```

Comment:
So, we're saying remove this. How tricky would it be to show using an endpoint instead?